bin/runtime/src/lib.rs

```diff
@@ -453,6 +478,8 @@ impl pallet_staking::Config for Runtime {
     type MaxNominatorRewardedPerValidator = MaxNominatorRewardedPerValidator;
     type OffendingValidatorsThreshold = OffendingValidatorsThreshold;
     type SortedListProvider = pallet_staking::UseNominatorsMap<Runtime>;
+    type MaxUnlockingChunks = ();
```

Comment:
It should definitely be sth else than `()`, eg. `ConstU32<16>`. Document this field [here](https://www.notion.so/cardinalcryptography/Staking-7a88dc9fb6dc44e085cbc7bf4e309193).

Reply:
I cannot edit this document, it's locked

Reply:
Done, I will edit the document in a second